examples/trusted_database/module_1/rust/src/lib.rs

```diff
@@ -0,0 +1,144 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Trusted Database example for Project Oak.
+//!
+//! This shows how an Oak Node can host an in-memory database and process private queries from
+//! clients.
+//!
+//! Current example implementation uses a list of Santander Cycles in London as test database:
+//! https://tfl.gov.uk/tfl/syndication/feeds/cycle-hire/livecyclehireupdates.xml
+//!
+//! Clients send their location coordinates (latitude and longitude) and the Oak Node returns the
+//! location of the closest Point Of Interest (in our case a bike station).
+
+pub mod proto {
+    pub mod oak {
+        pub use oak::proto::oak::invocation;
+        pub mod examples {
+            pub mod trusted_database {
+                include!(concat!(
+                    env!("OUT_DIR"),
+                    "/oak.examples.trusted_database.rs"
+                ));
+                include!(concat!(
+                    env!("OUT_DIR"),
+                    "/oak.examples.trusted_database_command.rs"
+                ));
+            }
+        }
+    }
+}
+
+// #[cfg(test)]
+// mod tests;
+
+use log::{debug, error};
+use oak::{grpc, io::ReceiverExt};
+use proto::oak::examples::trusted_database::{
+    ListPointsOfInterestRequest, ListPointsOfInterestResponse, Location, PointOfInterest,
+    TrustedDatabase, TrustedDatabaseCommand, TrustedDatabaseDispatcher,
+};
+
+/// Oak Node that contains an in-memory database of Points Of Interest.
```

Comment:
Is it intentional that the documentation is the same as the other node? That seems confusing, maybe add more details